src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -806,3 +809,20 @@ def __set_errors_json(self, error_count_by_operation, errors_by_operation):
         }
     # endregion
 
+    def __removed_older_complete_status_files(self, status_folder):
+        """ Retain 10 latest status complete file and remove other .complete.status files """
+        files_to_removed = []
+        complete_status_files_list = glob.glob(status_folder + '/' + '*.complete.status')   # Glob return empty list if no file matched pattern
```

Comment:
nit: rename to complete_status_file_list or all_complete_status_files

Reply:
done